### DISCUSSION TITLE
0.3.2 - Breaking changes (Please read release notes)

### DISCUSSION THREAD

**Discussion by firstof9** [moralmunky/Home-Assistant-Mail-And-Packages#447] [General]
[![Downloads for this release](https://img.shields.io/github/downloads/moralmunky/Home-Assistant-Mail-And-Packages/0.3.2/total.svg)](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/releases/0.3.2)

## :boom: Breaking Change :boom:

Images are now stored in the component's directory in images directory. ie: /config/custom_components/mail_and_packages/images/
If the `Create image for notification apps` option is checked, they will be **copied** to the `www/mail_and_packages` directory as well.

This creates camera entities automatically for USPS mail images and Amazon delivery images, no more having to make your own local file cameras.

## :zap: Enhancements :zap:
- refactor: config flow @firstof9 (#436)
- Add additional subject for FedEx @firstof9 (#419)
- Add 'no deliveries' image @firstof9 (#409)
- Rework DataUpdateCoordinator @firstof9 (#404)

## :sparkles: New Features :sparkles:
- feat: add UPS exception sensor @firstof9 (#426)
- Add amazon exception sensor @firstof9 (#420)
- Add Mail camera @firstof9 (#407)
- Add USPS Exception sensor @firstof9 (#386)

## 🐛  Bug Fixes 🐛 
- fix(shipper): fix usp_delivering sensor @firstof9 (#440)
- fix: filter empty list for amazon forwards @firstof9 (#434)
- fix: resolve bug in amazon exception sensor @firstof9 (#432)
- fix(shipper): update for new UPS formatted emails @firstof9 (#423)
- fix: Fix Amazon exception email address @firstof9 (#422)
- fix: wrong filename generated @firstof9 (#414)
- Fix default for DEFAULT_PATH and tests adjustments @firstof9 (#402)
- Strip quotes from Amazon forward emails @firstof9 (#406)
- Handle additional date formats in Amazon emails @firstof9 (#391)
- Better fix for config flow defaults @firstof9 (#385)

### :wrench: Maintenance :wrench:
- Fix links in README @firstof9 (#444)
- chore: update workflow configs @firstof9 (#437)
- chore: new labeler github action @firstof9 (#427)
- chore: adjust release-drafter @firstof9 (#428)

## Links
- [Wiki](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/wiki)
- [Submit bugs/feature requests](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/issues)

<hr /><em>This discussion was created from the release <a href='https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/releases/tag/0.3.2'>0.3.2 - Breaking changes (Please read release notes)</a>.</em>

**Comment by Adam-North** [comment]
Does this work with the latest beta release of the custom card?

**Reply by firstof9** [reply]
As far as I am aware, yes, the beta version of the card should work properly with this release.
